src/WebJobs.Script/ManagedDependencies/ManagedDependencyOptions.cs

```diff
@@ -1,10 +1,6 @@
 ﻿// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-using System.Collections.Generic;
-using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
-using Newtonsoft.Json;
-
```

Comment:
nit: undo as this is not related and required.

Reply:
Removing `using Microsoft.Azure.WebJobs.Script.Grpc.Messages;` is required, for separating the project. I removed others too that weren't being used. I can add them back.

Reply:
thanks..no need to add unsed statements back!